package.json

```diff
@@ -24,17 +27,20 @@
         "eslint-config-prettier": "^7.2.0",
         "eslint-formatter-pretty": "^4.0.0",
         "eslint-import-resolver-webpack": "^0.13.0",
-        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-import": "^2.23.4",
         "eslint-plugin-prettier": "^3.3.1",
         "eslint-plugin-react": "^7.22.0",
         "eslint-plugin-react-hooks": "^4.2.0",
         "husky": "^4.2.5",
         "jest": "^26.4.2",
         "lerna": "^3.22.0",
         "lint-staged": "^10.4.0",
+        "postcss": "^8.3.6",
         "prettier": "^2.1.2",
+        "react": "^16.14.0",
+        "react-dom": "^16.14.0",
```

Comment:
Shouldn't we add React to dependencies list instead?

Reply:
you are right, I just moved them to dependencies